### ISSUE TITLE
Cannot actually edit via the link at the bottom

### ISSUE THREAD

**Issue by ankon** [docker/docker.github.io#808]
### Problem description

> We really want your feedback, and we've made it easy. You can edit, rate, or file an issue at the bottom of every page on docs.docker.com.

I tried that from https://docs.docker.com/engine/reference/commandline/build/, which brings me to https://github.com/docker/docker.github.io/edit/master/engine/reference/commandline/build.md, which gives a 404 error.

FWIW: The actual problem I wanted to quickly fix is a missing section header for the `-f` option on that file.

**Comment by joaofnfernandes** [comment]
Thanks for reporting this! All the documentation is now tracked in this repository, but there are a couple of articles that are managed in an upstream repository, like the reference articles you were trying to edit.

We still need to improve this experience, but in the meantime you can submit a pull request to fix that article in the [docker repository](https://github.com/docker/docker/blob/master/docs/reference/commandline/build.md)

**Comment by mstanleyjones** [comment]
I just tried this and the `Edit this page` at the bottom does work for me (it opens the relevant page in the Github editor), but it behaves different from the `Edit this page` at the upper right side of the page, which just takes you to the right directory of the upstream repo. In fact, the one on the right-hand side is behaving correctly, because the `docker/docker` repository does not allow you to submit changes using the Github editor, since you need to save your commits.

So, long story short is that I'll change the behavior for the bottom link to match the behavior of the right-hand link, and when you get a result like this it means that the file in question is not in our control and that you need to use the processes for the particular upstream repo to get your changes in.

Hope that helps!
